lec06-pop-models.Rmd

```diff
@@ -2,3 +2,298 @@
 title: "Qualitative review of population models"
 ---
 
+
+## Lesson preamble
+
+In developing this series of lectures I have referred heavily to the notes from Nonlinear Physics,
+[PHY1460](https://www.physics.utoronto.ca/students/graduate-courses/current/phy1460s), 
+which I would encourage anyone who is interested in getting more in depth on the quantitative analysis 
+of dynamical systems to take or audit. I also refer to "Nonlinear Dynamics and Chaos" by Steven Strogatz, 
+a very good textbook which you might find helpful. You can find the pdf online, it's available as an e-book
+from the UofT library, and it's available in the Physics library as a hard copy course reserve.
+
+## Learning objectives:
+
+- Appreciate the purpose and usefulness of modelling real-world systems
+- Understand the use of differential equations to model populations
+- Qualitatively analyze one-dimensional population models by hand and in R
+    - Find fixed points graphically
+    - Analyze stability graphically
+    - Draw one-dimensional phase portraits
+
+## Lesson outline:
+
+Total lesson time: 2 hours
+
+- Introduction to modelling (20 min)
+- Setting up a model (40 min)
+- Qualitative analysis of one-dimensional models (60 min)
+
+----
+
+## Introduction - why model?
+
+*How many have taken some theoretical ecology before?*
+
+1. Models can simplify a complex system and help us identify what's important.
+  - biology, as we all know, is extremely complicated
+  - models summarize what we think we know about a system, 
+  force us to identify the important parameters and distill them down to a manageable number of parameters
+   
+   *Note*: that's not always how modelling is done. Sometimes you just add everything you can think of 
+   to your very complicated model, and you hope by doing that, your model will be more accurate 
+   and make better predictions. But to do that you need to simulate the model - it will likely 
+   be too complicated to solve analytically. (Analytical = a version you can leave with variables 
+   instead of putting numbers in. In R we will mostly be simulating.) An example of a discipline where 
```

Comment:
`simulating).`

Reply:
Disagree - the period only goes outside the parentheses if the parentheses begin inside a sentence. If the entire sentence is in parentheses, the period goes inside :)

Reply:
Didn't know that! Neat!